tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:latest
+FROM rust:1.34.0
```

Comment:
This makes sense. `:latest` will always grab the latest version and would invalidate the tests so this is legit 👍